practices/team_roles_definition.md

```diff
@@ -0,0 +1,103 @@
+    Example: Director  
+    
+**2. Resource Investigator**  
+    Explores opportunities, handles external contact.  
+    Example: [Product Owner](/practices/product-owner)  
+    
+**3. Coordinator/Chairman**  
+    Clarifies goals, coordinates resources.  
+    Example: Project Manager  
+    
+**4. Shaper**  
+	An alternative to a coordinator. Gives coherence to team work.  
+  
+**5. Monitor/Evaluator**  
+	Evaluates ideas, looks for different perspectives.  
+	Example: Test Analyst, Tester  
+  
+**6. Teamworker**  
+	Builds bridges, takes care of the team spirit  
+	Example: [Team Lead](/practices/team-lead), Scrum Master  
+  
+**7. Implementer**  
+  Pursues the plans into practice.  
+  Example: Developer  
+  
+**8. Completer/Finisher**  
+	Searches for errors.  
+	Example: Tester, QA Manager  
+  
+**9. Specialist**  
+	Provides scarce knowledge and skills.  
+	Example: Senior Developer  
+
+![Belbin Team Roles](/files/team_roles_definition.png)
+[Belbin Team Roles](https://www.prepearl.net/belbin-team-roles/)
+
+## Why You Might Want the Team Roles
+
+Belbin recognized that the team success depends on the behavior of the team members. Defining various team roles allows you to match job requirements with the appropriate skills and  therefore, you can create a well-balanced team. Belbin’s research shows that the team tends to work more cohesively if the team members understand their individual roles. Familiarity leads to better performance, sharing information engages team members to find constructive solutions and ideas. It reduces misunderstandings and disputes. Finding a good match between the team members and their roles eliminates mistakes and improve quality of work.
+
+## Issues the Team Roles Solve
+
+- [Demotivated team](/issues/demotivated-team)
+- ["Not my problem" mentality](/issues/not-my-problem-mentality)
+- Unsuccessful product
+- Unhappy clients
+- Toxic team culture
+
+## How to Define the Team Roles
+
+**Maturing team roles**  
+  Team roles and their responsibilities are not always necessarily static. Team members can adopt different roles at different times, team roles develop and mature.  
+
+**Hiring**  
+  It is important to define the roles you need in your team and then recruit, hire, train, and retain the right people. Define the roles before hiring! Look into the applicants’ qualifications, characteristics, strengths, and weeknesses. Explain clearly their responsibilities to them.  
+
+**Understanding**  
+  Each team member also has to know their superiors, corresponding expectations, experience and educational requirements, team goals, and how they will be evaluated.  
+
+**Chart**  
+  It is a good idea to create a chart once you are finished with creating the team roles. The team members would understand the relationships easier if they can look at it.   
+
+## Common Pitfalls of the Team Roles
+
+Find the common pitfalls in the  Belbin chart above.  
```

Comment:
Please add some Common Pitfalls here. We don't want to move the user outside of our website. So we shouldn't offer him another site here.

Reply:
done